docs/admin/logging/gcplogs.md

```diff
@@ -0,0 +1,63 @@
+<!--[metadata]>
++++
+aliases = ["/engine/reference/logging/gcplogs/"]
```

Comment:
you can remove this line, because this is a new page to the documentation, so we don't need a redirect/alias for an old location